NAMESPACE

```diff
@@ -1,5 +1,6 @@
 # Generated by roxygen2 (4.1.1): do not edit by hand
 
+S3method(all,equal.transformation)
```

Comment:
@robertzk FYI `NAMESPACE` was generated incorrectly.  You need to specify `#' @method all.equal transformation` in your roxygen.